src/tiled/commanddialog.cpp

```diff
@@ -63,6 +69,24 @@ void CommandDialog::closeEvent(QCloseEvent *event)
     CommandManager::instance()->updateActions();
 }
 
+void CommandDialog::setShortcut(const QKeySequence &keySequence)
+{
+    if (!mUi->keySequenceEdit->isEnabled())
+        mUi->keySequenceEdit->setEnabled(true);
```

Comment:
Is it okay to call public slots directly like functions? I mean, syntactically it's correct, but in Qt style, is it considered correct? I don't know why I get this feeling that there is some better way to do it rather than call a public slot directly :confused: